Sources/TransferMethod/SelectTransferMethodTypeCell.swift

```diff
@@ -44,47 +39,52 @@ final class SelectTransferMethodTypeCell: UITableViewCell {
 
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
-        iconView.backgroundColor = Theme.Icon.backgroundColor
+        imageView?.backgroundColor = Theme.Icon.backgroundColor
     }
 
     override func setHighlighted(_ highlighted: Bool, animated: Bool) {
         super.setHighlighted(highlighted, animated: animated)
-        iconView.backgroundColor = Theme.Icon.backgroundColor
+        imageView?.backgroundColor = Theme.Icon.backgroundColor
     }
 
     // MARK: Theme manager's proxy properties
     @objc dynamic var titleLabelColor: UIColor! {
-        get { return self.titleLabel.textColor }
-        set { self.titleLabel.textColor = newValue }
+        get { return self.textLabel?.textColor }
+        set { self.textLabel?.textColor = newValue }
     }
 
     @objc dynamic var titleLabelFont: UIFont! {
-        get { return self.titleLabel.font }
-        set { self.titleLabel.font = newValue }
+        get { return self.textLabel?.font }
+        set { self.textLabel?.font = newValue }
     }
 
     @objc dynamic var descriptionLabelColor: UIColor! {
```

Comment:
We are not using this property. Could you remove?

Reply:
Sure